client/my-sites/checkout/checkout-thank-you/redesign-v2/sections/footer/style.scss

```diff
@@ -129,4 +127,162 @@
 			gap: 40px;
 		}
 	}
+
+	.checkout-thank-you__footer-professional-email {
```

Comment:
In additional to https://github.com/Automattic/wp-calypso/pull/82293/files#r1339492903, might also be a good chance to break down the CSS files, try to keep things a bit more contained. I know right now even with redesign-v2, we're still trying to override a bunch of stuff, but at least for the new things, we can try to clean it up a bit.